crates/floresta-wire/src/p2p_wire/running_node.rs

```diff
@@ -702,7 +704,7 @@ where
                 &block.block.txdata,
                 &self.chain,
             )?;
-            async_std::task::yield_now().await;
+            tokio::task::yield_now().await;
```

Comment:
This never helped with the problem it attempted to solve. You can just remove it.